steely/plugins/btc.py

```diff
@@ -0,0 +1,28 @@
+'''Get the latest Bitcoin price (in USD and EUR)'''
+import requests
+
+__author__ = 'CianLR'
+COMMAND = 'btc'
+
+API_URL = 'https://api.coindesk.com/v1/bpi/currentprice.json'
+RESP_TEMPLATE = """Bitcoin Price:
+    €{}
+    ${}"""
+
+def get_btc():
+    response = requests.get(API_URL)
+    if not response.ok:
+        return "Error getting price. Response (code {}) {}".format(
+            response.status_code, response.text)
+    j = response.json()
+    if 'bpi' not in j:
+        return "Prices missing from JSON: {}".format(str(j))
+    return RESP_TEMPLATE.format(j['bpi']['EUR']['rate'],
```

Comment:
You should probably also check if EUR and rate are in the json, just to be on the safe side and prevent hidden errors (ahem linden)

Reply:
Okaaaay, damn boilerplate.